src/easy-peasy.js

```diff
@@ -4,15 +4,25 @@ import {
   createStore as reduxCreateStore,
 } from 'redux'
 import memoizeOne from 'memoize-one'
-import produce from 'immer'
+import produce, { setAutoFreeze } from 'immer'
 import thunk from 'redux-thunk'
 import { isStateObject } from './lib'
 
-const effectSymbol = '__effect__'
-const selectSymbol = '__select__'
-const selectDependenciesSymbol = '__selectDependencies__'
-const selectStateSymbol = '__selectState__'
-const reducerSymbol = '__reducer__'
+setAutoFreeze(false)
+
+const effectSymbol = Symbol('__effect__')
```

Comment:
We had issues with Symbols - see #16

Reply:
Ah ok, maybe you could take the same approach that I took with reducerContainer, a wrapper and then instanceof, to let clean the original references. I think adding property, and especially this way, to parameters will polute them.